fastlane_core/lib/fastlane_core/crash_reporter/backtrace_sanitizer.rb

```diff
@@ -0,0 +1,49 @@
+module FastlaneCore
+  class BacktraceSanitizer
+
+    def self.sanitize(type: nil, backtrace: nil)
+      if type == :user_error || type == :crash
+        # If the crash is from `UI` we only want to include the stack trace
+        # up to the point where the crash was initiated.
+        # The two stack frames we are dropping are `method_missing` and
+        # the call to `crash!` or `user_error!`.
```

Comment:
Yeah, it just causes the crashes to have the same name, they are not necessarily grouped together because the stack traces are different, but they are hard to disambiguate.